server/handlers/subscription_handler.coffee

```diff
@@ -281,8 +281,10 @@ class SubscriptionHandler extends Handler
 
     else
       couponID = user.get('stripe')?.couponID
-      if user.get('country') is 'brazil'
-        couponID ?= 'brazil'
+      if not couponID and user.get 'country'
+        basicProduct = yield Product.findBasicSubscriptionForUser user
+        if basicProduct.name isnt 'basic_subscription'  # We have a customized product for this country
+          couponID = user.get 'country'
```

Comment:
I'm confused about using the country property as a couponID. What's the effect of that? Should the two be conflated?